cea/default.config

```diff
@@ -767,30 +767,30 @@ output-path = {general:scenario}/../..
 output-path.type = PathParameter
 output-path.help = Path to location where to store the new project
 
-[disconnected-cooling]
+[decentralized]
```

Comment:
really not a fan of this name. decentralized what? i know, it's consistent with the name of the tool. which is good. let's think about a good name for the tool, though.